packages/tasks/src/tasks/image-to-image/about.md

```diff
@@ -1,24 +1,23 @@
 ## Use Cases
 
-### Style transfer
-
-One of the most popular use cases of image-to-image is style transfer. Style transfer models can convert a normal photography into a painting in the style of a famous painter.
-
-## Task Variants
-
 ### Image inpainting
 
-Image inpainting is widely used during photography editing to remove unwanted objects, such as poles, wires, or sensor
-dust.
+Image inpainting is widely used during photography editing to remove unwanted objects, such as poles, wires, or sensor dust.
+
+- check out & learn more: [inpainting demo](https://huggingface.co/spaces/diffusers/stable-diffusion-xl-inpainting), [image inpainting with 🧨diffusers](https://huggingface.co/docs/diffusers/main/en/api/pipelines/stable_diffusion/inpaint)
```

Comment:
We do not want to redirect people to another page mid-page, it increases bounce rate and undermines SEO. you can put it at the end